Vagrantfile

```diff
@@ -36,6 +36,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.provider "virtualbox" do |vbox|
     vbox.cpus = 4
-    vbox.memory = 4096
+    vbox.memory = 8192
```

Comment:
Any particular reason we're doubling the memory? Just curious. 😄

Reply:
It was part of my JVM-dumping-core testing, but not a bad minimum for Dataverse, particularly when running integration tests, so I kept it.